### DISCUSSION TITLE
Disable unresolved references diagnostic

### DISCUSSION THREAD

**Discussion by ChristinWhite** [epwalsh/obsidian.nvim#630] [Q&A]
Is it possible to disable just the 'Unresolved Reference' diagnostic info flags selectively? I use `[[Note I intend to create at some point]]` links pretty frequently and it triggers a lot of extra noise when working with my Obsidian vault. 

I didn't see anything about configuring individual diagnostic rules in the documentation but I may have missed it, if you could point me in the right direction it would be appreciated!

I'm so excited about using Neovim with this plugin to complement my use of Obsidian, I really appreciate all the hard work that's clearly gone into this plugin! It's the perfect complement to Obsidian.

**Comment by epwalsh** [comment]
Obsidian.nvim is not a language server, so those diagnostics are coming from a different plugin.

**Comment by ChristinWhite** [comment]
Thanks, I thought I'd disabled an LSP that I hadn't and got it confused with this, sorry for the confusion!
